docs/getting-started.md

```diff
@@ -24,22 +24,22 @@ npx react-native init <projectName> --template react-native@^0.63.2
 
 Once your project has been initialized, React Native will have created a new sub directory where all your generated files live.
 
-```
+```cmd
 cd <projectName>
 ```
 
 ### Install the Windows extension
 
 Lastly, install the React Native for Windows packages.
 
-```
+```cmd
 npx react-native-windows-init --overwrite
 ```
 
 > The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect. We are tracking [this issue here](https://github.com/microsoft/react-native-windows/issues/4698).
 
 Here are the options that `react-native-windows-init` takes:
-```none
+```bash

```

Comment:
this should probably stay as none (it's just text output from running the script with `--help`), or better yet convert it into a markdown table instead :)

Reply:
The website seems to want to highlight random sections of the table with ```none. I'll attach a photo below.  Maybe I'll just go ahead and convert it to a table then.
<img width="549" alt="Capture2" src="https://user-images.githubusercontent.com/34109996/92183854-6d583a80-ee04-11ea-89f7-541f85354b78.PNG">

Reply:
Ah yeah, that looks wonky, thanks for fixing it @chiaramooney !
FYI here is the list of syntaxes/languages that GitHub understands:
https://github.com/github/linguist/blob/master/vendor/README.md

Note that `cmd` isn't there sadly but powershell is so it should be ok to use instead for simple commands like `cd`

Reply:
> FYI here is the list of syntaxes/languages that GitHub understands:
> https://github.com/github/linguist/blob/master/vendor/README.md

Do we know what the RN website style supports? Is it based on the same core as GitHub's markdown rendering, or something else?

Reply:
@chrisglein So it's actually more complicated than just GitHub's markdown (of course...). We use Docusaurus to produce the content from the markdown files, which I believe RN website does too, so the list I posted earlier might not be 100% a match to Docusaurus.

GitHub does have cmd support but it isn't called cmd which makes it hard to find so I originally missed it. It's called Batchfile and it has a few aliases you can use (instead of cmd): `bat`, `batch`, `dosbatch`, `winbatch`.

https://github.com/github/linguist/blob/b410c72001e5023f9fdc9f0d601713878a167c90/lib/linguist/languages.yml#L439

The best way for @chiaramooney to test this is to clone the website repo, make the changes locally, run `yarn build` and `yarn start` locally then navigate to the docs (under /docs/next) and see how it is rendered.

Reply:
Sounds good I'll try this today.